### DISCUSSION TITLE
vagrant zsh completion

### DISCUSSION THREAD

**Discussion by nhojpatrick** [Homebrew/discussions#236] [Everyday usage]
Before I reinvent the wheel does anyone have auto completion like https://formulae.brew.sh/formula/vagrant-completion for zsh working?

I can find a few vagrant zsh autocompletions but none setup via `brew install xxx`.

Happy setup if not, but doesn't anyone have any tips of which forumla to follow???

**Comment by SMillerDev** [comment]
You can make a pull request to add ZSH support to the formula, see: https://docs.brew.sh/Updating-Software-in-Homebrew

**Comment by nhojpatrick** [comment]
@SMillerDev I missed this reply, do you know of an example zsh autocompletion I can use as a template that homebrew already uses?

**Reply by gromgit** [reply]
Check out the `ninja` formula. Completions are installed alongside other Zsh function libraries in `$HOMEBREW_PREFIX/share/zsh/site-functions`.

**Comment by J-M0** [comment]
Vagrant has official zsh completions. I was going to submit a PR to update `vagrant-completion`, but then I noticed that it would conflict with `zsh-completions`.
